app/assets/stylesheets/_action-sheet.scss

```diff
@@ -104,3 +104,9 @@
     margin-top: 0.5em;
   }
 }
+
+.field_with_errors {
```

Comment:
Selector `field_with_errors` should be written in hyphenated BEM (Block Element Modifier) format